### PR TITLE
Update info.json

### PR DESCRIPTION
**Pull request by PaladinFallen**

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "Exoskeleton Mark II",
-  "version": "3.0.0",
+  "version": "3.1.0",
```
Comment (axtox) [inline info.json]: the versioning guideline tells that if the application was changed in the way that it becomes incompatible with previus version, you need you change the major number. This is our case, so I suggest to change version to 4.0.0

**Review by axtox — CHANGES_REQUESTED** [review]
See comments in 'files' section. Make changes in order to merge this pull request

**Comment by PaladinFallen** [comment]
I made the fixes to the files, so everything is good.

**Review by axtox — APPROVED** [review]
Okay, now it aligned with guidelines! Thank you!

**Pull request merged**